.github/workflows/release-and-publish-artifacts.yml

```diff
@@ -0,0 +1,33 @@
+name: release and push to central
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup build dependencies
+        run: make setup
+      - name: Set up Java for publishing to Maven Central Repository
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
```

Comment:
Is this okay? Docs on the action indicate `8` is an acceptable value, but not the old school `1.8` version format.

Reply:
Will update.

Reply:
See also https://github.com/actions/setup-java/blob/main/docs/switching-to-v2.md

Reply:
Looks like v1 supported the legacy version syntax, while v2 does not. This should be fixed.